std/range/package.d

```diff
@@ -1707,7 +1707,7 @@ range. Iteration spans the entire range.
 auto radial(Range, I)(Range r, I startingIndex)
 if (isRandomAccessRange!(Unqual!Range) && hasLength!(Unqual!Range) && isIntegral!I)
```

Comment:
Well, that's a separate change and also requires a test. How about you do that?
And it should be `hasSlicing!(Unqual!Range) && isBidirectionalRange!(Unqual!Range)`.